example/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_REST_BASE_URL=
```

Comment:
Is this file necessary?

Reply:
No.  I thought it might serve as an example but the README is probably sufficient.  I've removed it.